docs/release_notes.md

```diff
@@ -1,5 +1,43 @@
 #
 
+## 5.0.2
+
+Release Date: 2025-09-XX
+
+The iRODS Consortium and RENCI are pleased to announce iRODS 5.0.2.
+
+This patch release adds support for Enterprise Linux 10 and Debian 13. With that comes bug fixes and improvements for `irsync`, GenQuery2, unattended installation, physical quotas, etc.
+
+This release consists of [30 commits from 9 contributors](https://github.com/irods/irods/compare/5.0.1...5.0.2).
+
+The latest binary packages for RockyLinux9, RockyLinux10, Ubuntu22, Ubuntu24, Debian12, and Debian13 are available at <https://packages.irods.org/>.
```

Comment:
Hmm, rockylinux or enterprise linux?

Reply:
For release testing, we'll lock in the OS for each version of EL. For example, almalinux 9 for EL9 and rockylinux 10 for EL10.

@SwooshyCueb Any thoughts on this conversation?

Reply:
9 has only been rocky i believe.